terraform/examples/io500/setup_io500.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# Load needed variables
+source ./configure.sh
+
+# Clean and configure DAOS servers
+source ./clean.sh
+
+echo "Copy agent config files from server"
+rm -f .ssh/known_hosts
+scp ${DAOS_FIRST_SERVER}:/etc/daos/daos_agent.yml .
+scp ${DAOS_FIRST_SERVER}:/etc/daos/daos_control.yml .
+
+echo "Configure DAOS Clients"
+for client in ${CLIENTS}
+do
+    echo "##################################################"
+    echo "#  ${client}"
+    echo "##################################################"
+    ssh ${client} "rm -f .ssh/known_hosts"
+    ssh ${client} "sudo systemctl stop daos_agent"
+    scp daos_agent.yml daos_control.yml ${client}:~
+    ssh ${client} "sudo cp daos_agent.yml daos_control.yml /etc/daos/"
+    ssh ${client} "sudo systemctl start daos_agent"
+done
```

Comment:
Maybe we could use pdsh or even better clush (see clustershell) for this? We could fix this after the patch is landed through.